netbox/ipam/models.py

```diff
@@ -497,7 +497,8 @@ class Meta:
         verbose_name_plural = 'VLAN groups'
 
     def __str__(self):
-        return u'{} - {}'.format(self.site.name, self.name)
+        site_name = self.site.name if self.site else '__global'
```

Comment:
`__str__` should return something user-friendly. We can simply omit the site name for global VLANs.